README.md

```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
 We cover material in logical order, so that if you do read the earlier sections first,
 later sections on the interactions of tax and compensation will be clearer.
 
-The first sections are about how companies work and how they grow:
+We start with [**Equity Compensation Basics**](#equity-compensation-basics): What compensation
+and equity compensation are.
+But before we get much further, we need to talk about what stock is, and how companies are
+formed.
 
-- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): To understand
-  exactly what equity is, it’s essential to know how companies organize their ownership, how
-  stock works, and how ownership typically evolves.
-- [**Startups and Growth**](#startups-and-growth): While not everyone reading this works at an early
-  stage company, those who do can benefit from understanding the particulars of how these
-  companies raise money and grow.
+[**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations) covers how companies
+organize their ownership, how stock is issued, public and private companies, and IPOs and
+liquidity.
 
-Then compensation:
+While not everyone reading this works at an early stage company, those who do can benefit
+from understanding the particulars of [**Startups and Growth**](#startups-and-growth).
 
-- [**Equity Compensation Basics**](#equity-compensation-basics): Here we dive into the fundamentals
-  of compensation and when and whether you can sell equity for cash.
-  These are important concepts to learn in order to understand what equity might be worth.
-- [**How Equity is Granted**](#how-equity-is-granted): This section covers the forms in which equity
-  is most commonly granted, including restricted stock grants, stock options, and RSUs.
+Finally the meat of our discussion: [**How Equity is Granted**](#how-equity-is-granted). This
```

Comment:
Elide the meat. 🥩 (Consider keeping the structure of the roadmap intact--just the chapter titles and descriptions, no intros. Ditto "We start with" at the top. Not needed!)

Reply:
(Readers don't expect interruptions before chapter titles in a TOC like this. Well enough without.)

Reply:
(Also, the roadmap is something we'll want as regular as possible across Guides, so simplifying it structurally will help with expectations imo.)